### DISCUSSION TITLE
Can inline code text be copied to clipboard?

### DISCUSSION THREAD

**Discussion by sebaslavigne** [squidfunk/mkdocs-material#7788] [Q&A]
There's an option to have code blocks have a button on the corner to copy the text to clipboard. I've noticed that the icon search in the documentation (https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#search) displays the results as inline code that can be clicked to copy the values to the clipboard.

In the page html these result as being `<button class="md-clipboard--inline" ... <code> ...`

Can this be replicated somehow?

**Comment by kamilkrzyskow** [comment]
Hello @sebaslavigne,
I guess you're looking for this discussion, this can be achieved with customization:
- https://github.com/squidfunk/mkdocs-material/discussions/6393

The `data-clipboard-text` signifies that the https://clipboardjs.com/ should handle the copy logic for it, there are also other attributes / selectors you could look over on that page.
